classes/PodsData.php

```diff
@@ -1806,7 +1799,7 @@ public static function table_create( $table, $fields, $if_not_exists = false ) {
 			$sql .= ' IF NOT EXISTS';
 		}
 
-		$sql .= " `{$wpdb->prefix}" . self::$prefix . "{$table}` ({$fields})";
+		$sql .= " `" . self::get_pods_prefix() . "{$table}` ({$fields})";
```

Comment:
:no_entry_sign: String " `" does not require double quotes; use single quotes instead

<sub>hash: 6b11fac6f85f2b72ea41dad1a52cbba5</sub>